README.md

```diff
@@ -0,0 +1,57 @@
+# Software Developers Global Summit 2020
+Annual conference for developers around the world.
+
+## Project Design:
+
+![screenshot](idesign/images/page-screenshot.PNG)
+
+This project was developed following the [CC Global Summit 2015](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) design by [Cindy Shin](https://www.behance.net/adagio07). It is a Microverse capstone project that I developed using bootstrap and a few more technologies (mentioned below). 
+
+Software developers Global Summit 2020 is a conference held at the end of each year. The site consist of three (Home, About, & Tickets) pages which do not have any functionality yet. The goal is to follow the given ([CC Global Summit 2015](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)) design, & personalise the content to anything relating to conferencing. You can see the project live demo (link provided below). Each of the pages on the site has got the mobile screen version (max-width ==> 768px) & desktop version (min-width ==> 768px).
```

Comment:
- [ ] Also you should add a video which you and your partner should describe the project in the video

![n ns](https://user-images.githubusercontent.com/61562626/132517797-c8b497bc-950e-4bfb-94f3-72037bde4a7c.jpeg)

Reply:
> * [x]  Also you should add a video which you and your partner should describe the project in the video
> 
> ![n ns](https://user-images.githubusercontent.com/61562626/132517797-c8b497bc-950e-4bfb-94f3-72037bde4a7c.jpeg)

- [ ] This issue is not addressed yet. I cannot find the video documentation link in the readme or PR summary. As the previous reviewer already pointed out you should prepare a video documentation and provide the link in the readme file. :+1:

Reply:
I need help with this I dont know how to do it